test/images/image-util.sh

```diff
@@ -35,7 +35,7 @@ listArchs() {
 # Returns baseimage need to used in Dockerfile for any given architecture
 getBaseImage() {
   arch=$1
-  echo $(grep "${arch}=" BASEIMAGE | cut -d= -f2)
+  grep "${arch}=" BASEIMAGE | cut -d= -f2
```

Comment:
```
In ./test/images/image-util.sh line 38:
  echo $(grep "${arch}=" BASEIMAGE | cut -d= -f2)
       ^-- SC2046: Quote this to prevent word splitting.
       ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
```